### PR TITLE
explicit random seed

### PR DESCRIPTION
**Pull request by mmoskal**

### CODE REVIEW

File: libs/base/eventcontext.ts [hunk 1]
```diff
@@ -77,7 +83,12 @@ namespace control {
             control.enablePerfCounter("all frame callbacks")
 
             let loopStart = control.millis()
-            this.deltaTimeMillis = loopStart - this.prevTimeMillis;
+            if (_animationTime) {
+                this.deltaTimeMillis = 30;
```
Comment (pelikhan) [inline libs/base/eventcontext.ts]: What’s the meaning of 30?

Comment (mmoskal) [reply libs/base/eventcontext.ts]: This is what we typically get without this (ie 31 fps or so)

**Comment by pelikhan** [comment]
Do we want this one? @mmoskal

**Comment by mmoskal** [comment]
It didn't really make it deterministic, so I think let's not complicate.

**Pull request closed without merging**